modules/sandboxed-containers-viewing-logs.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/troubleshooting-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-view-debug-logs_{context}"]
+= Viewing debug logs for {sandboxed-containers-first}
+
+Cluster administrators can use the enhanced debug logs for {sandboxed-containers-first} to troubleshoot issues. The logs for each node are printed to the node journal.
+
+You can review the logs for the following {sandboxed-containers-first} components:
+
+* Kata agent
+* Kata runtime (`containerd-shim-kata-v2`)
+* virtiofsd
+
+Logs for QEMU do not print to the node journal. However, a QEMU failure is reported to the runtime, and the console of the QEMU guest is printed to the node journal.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* To review the Kata agent logs, run:
```

Comment:
kata agent print its logs to the guest console, the guest console is printed by running this command, maybe worth describing the procedure mentioning this is for "kata-agent and guest console logs" (as we mentioned above qemu guest console is printed to node's journal but don't describe how not pointing to this procedure)

Reply:
Add to the kata agent logs command that this is also to view the guest console logs